client/src/config/keycloakConfig.ts

```diff
@@ -0,0 +1,15 @@
+const token = localStorage.getItem('token');
```

Comment:
My prefference for the config will be to be json as it is originally JSON and on openshift we will be able to simply override that by config-map if needed